packages/clerk-js/src/ui/ImpersonationFab/ImpersonationFab.tsx

```diff
@@ -101,6 +102,11 @@ export const ImpersonationFab = () => {
     return null;
   }
 
+  const title = localizationKeys('impersonationFab.title', {
+    identifier: getFullName(session.user) || getIdentifier(session.user),
+  });
+  const titleLength = useLocalizations().t(title).length;
```

Comment:
🛑 This is against [the rules of hooks](https://reactjs.org/docs/hooks-rules.html), as you should not have a hook conditionally executed - line 102 includes an early return statement. Let's move it near the top of the component definition: 

```
  const { t } = useLocalizations();
...
  if (!isImpersonating || !session.user) {
    return null;
  }
...
  const titleLength = t(title).length;
```

Can you check if eslint throws an error/warning for this one here? Thanks!

Reply:
Right! My bad! I missed that.

Reply:
✔️ fixed